lms/djangoapps/certificates/api.py

```diff
@@ -198,14 +198,16 @@ def get_recently_modified_certificates(course_keys=None, start_date=None, end_da
     return GeneratedCertificate.objects.filter(**cert_filter_args).order_by('modified_date')
 
 
+# lint-amnesty, pylint: disable=unused-argument
```

Comment:
Temporary. This will be cleaned up in an upcoming PR. This separates removing the `course` argument from all functions in the `certificates` app. The next PR will handle this in external django apps that use the `generate_user_certificates` or the `regenerate_user_certificates` function.